test/tests/mortar/convergence-studies/solution-continuity/tests

```diff
@@ -40,5 +44,6 @@
     requirement = 'The system shall be able to demonstrate asymptotically correct convergence rates of 2 and 1 for a mortar solution continuity problem with an equal geometric discretization when using a first order basis for the temperature variable and a zeroth order basis for the Lagrange multiplier.'
     method = '!dbg'
     required_python_packages = 'sympy pandas matplotlib'
+    compiler = 'gcc'
```

Comment:
Other compilers do not work for this for some reason?

Reply:
With our Test Clang target I get an Intel MKL failure. I suspect it's due to zero pivots or something since this is saddle point

Reply:
Could you point me to error messages? I was wondering if we are  able to fix that using some simple option?

Reply:
https://civet.inl.gov/job/603564/

Reply:
```
mortar/convergence-studies/solution-continuity.p2p1: [0]PETSC ERROR: See http://www.mcs.anl.gov/petsc/documentation/linearsolvertable.html for possible LU and Cholesky solvers
mortar/convergence-studies/solution-continuity.p2p1: [0]PETSC ERROR: Could not locate solver package mumps. Perhaps you must ./configure with --download-mumps
```

Could you add "-pc_factor_mat_solver_type superlu_dist" ?

Reply:
Oh lol. Should have read that message more closely. We should download mumps for that test target. WTF

Reply:
I get zero pivot with `superlu_dist`, even with `-pc_factor_shift_type NONZERO -pc_factor_shift_amount 1`. I believe this is why I picked MUMPS